core/src/main/java/org/frankframework/processors/InputOutputPipeProcessor.java

```diff
@@ -119,10 +117,10 @@ protected PipeRunResult processPipe(@Nonnull PipeLine pipeLine, @Nonnull IPipe p
 			if (pipe.isRestoreMovedElements()) {
 				log.debug("Pipeline of adapter [{}] restoring from compacted result for pipe [{}]", owner::getName, pipe::getName);
 				Message result = pipeRunResult.getResult();
-				if (!result.isEmpty()) {
+				if (result != null && !result.isEmpty()) {
```

Comment:
Misschien is `!Message.isEmpty(..)` iets mooier. Maar dit kan ook

Reply:
Eens. Ik dacht dat die alleen als member methode bestond, maar bij nader inzien zie ik dat die ook als static beschikbaar is.